WindowState/README.md

```diff
@@ -6,104 +6,213 @@ The component provides the current window information as a Compose state, includ
 
 ![postures](screenshots/postures-overview.png)
 
-And the window size classes are measured based on Google [Window size classes](https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes#window_size_classes) to help you support different screen sizes of the devices, including large screen and regular single screen devices.
+And the window size classes are measured based on Google's [Window size classes](https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes#window_size_classes) to help you support different screen sizes of devices, including large screen and regular single screen devices.
+
+## Add to your project
+
+1. Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:
+
+    ```gradle
+    allprojects {
+        repositories {
+            google()
+            mavenCentral()
+         }
+    }
+    ```
+
+2. Add dependencies to the module-level **build.gradle** file (current version may be different from what's shown here).
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen:windowstate:1.0.0-alpha02"
+    ```
+
+3. Also ensure the compileSdkVersion and targetSdkVersion are set to API 31 or newer in the module-level build.gradle file.
+
+    ```gradle
+    android { 
+        compileSdkVersion 31
+        
+        defaultConfig { 
+            targetSdkVersion 31
+        } 
+        ... 
+    }
+    ```
+
+4. Access the info about the window state from **WindowState** to build or adjust your UI. Please refer to the [sample](https://github.com/microsoft/surface-duo-compose-sdk/tree/main/WindowState/sample) for more details.
+
+## API reference
 
 ```kotlin
 @Composable
 fun Activity.rememberWindowState(): WindowState
 ```
+
 An interface to provide all the relevant info about the device window.
 
+### Fold properties
+
+```kotlin
+val foldSizeDp: Dp
+```
+
+Returns a dp value of the thickness of the hinge of dual-screen device or the folding line of foldable device when the device is in dual-screen mode. If the device is in single screen mode, or the device is a regular single screen device, the return value will be 0.
+
+```kotlin
+val foldablePane1SizeDp: Size
```

Comment:
Can we move `pane1SizeDp` and `pane2SizeDp` before `foldablePane1SizeDp` and `foldablePane2SizeDp`?